examples/rbfe_edge_list.py

```diff
@@ -0,0 +1,134 @@
+import argparse
+import csv
+import pickle
+
+import numpy as np
+from rdkit import Chem
+
+Chem.SetDefaultPickleProperties(Chem.PropertyPickleOptions.AllProps)
+
+from timemachine.constants import KCAL_TO_KJ
+from timemachine.fe import atom_mapping
+from timemachine.fe.rbfe import HostConfig, estimate_relative_free_energy
+from timemachine.fe.utils import get_mol_name
+from timemachine.ff import Forcefield
+from timemachine.md import builders
+from timemachine.parallel.client import CUDAPoolClient
+
+
+def get_mol_by_name(mols, name):
+    for m in mols:
+        if m.GetProp("_Name") == name:
```

Comment:
nit: use `get_mol_name`?

Reply:
Aside: a pattern I've found useful is something like
```python
mol_by_name = {get_mol_name(mol): mol for mol in mols}
# …
mol_a = mol_by_name[mol_a_name]
```

Reply:
willfix